src/Command/BaseCommand.php

```diff
@@ -43,6 +43,8 @@ protected function initialize(InputInterface $input, OutputInterface $output)
             $output->writeln('<info>Using configuration:</info> ' . $this->configurationFile);
             $output->writeln('<info>Changed working dir:</info> ' . $this->workingDir);
         }
+
+        $this->conductor->setOutput($output);
```

Comment:
Without knowing the base code this seems hacky. What is the reason for this line?